### PR TITLE
bump Go to v1.13.5

### PR DESCRIPTION
**Pull request by mar1n3r0**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12.5 AS buildenv
+FROM golang:1.13.5 AS buildenv
```
Comment (vardius) [inline Dockerfile]: Can you change to 1.13 so the latest 1.13.* is pulled ?

**Review by vardius — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (mar1n3r0) [reply Dockerfile]: Sure thing.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12.5 AS buildenv
+FROM golang:1.13.* AS buildenv
```
Comment (vardius) [inline Dockerfile]: Instead of 1.13.* keep 1.13 this way it pulls lates 1.13.*

File: go.mod [hunk 3]
```diff
@@ -1,6 +1,6 @@
 module github.com/vardius/pubsub
 
-go 1.13
+go 1.13.5
```
Comment (vardius) [inline go.mod]: same here

**Review by vardius — CHANGES_REQUESTED** [review]

**Pull request merged**